gravitee-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/AnalyticsServiceImpl.java

```diff
@@ -321,7 +321,6 @@ private TopHitsAnalytics convert(GroupByResponse groupByResponse) {
             metadata.put("version", apiEntity.getVersion());
         } catch (ApiNotFoundException anfe) {
             metadata.put("deleted", "true");
-            metadata.put("name", "Deleted API");
```

Comment:
Hum... not sure that is a good idea...
It would be probably better if we are using an other static code for unmapped API...

WDYT ?

Reply:
Hi, David!
Had same thoughts. Even started adding config to .yml-s and default _UNDEFINED_API const changes but then found some changes already in master (you've added those constants to API Management, with correct metadata fill in) and reverted all the things. This line is removed because of the code further - as it is useless actually.
Do you think it should be done in this PR?